pages/admin/permit-holders.tsx

```diff
@@ -273,6 +187,148 @@ export default function PermitHolders() {
   ];
   const userStatusOptions = [UserStatus.Active, UserStatus.Inactive];
 
+  // Set Permit Holder Inactive/Active modal state
+  const {
+    isOpen: isSetPermitHolderStatusModalOpen,
+    onOpen: onOpenSetPermitHolderStatusModal,
+    onClose: onCloseSetPermitHolderStatusModal,
+  } = useDisclosure();
+
+  // Sets the data required for the Set Permit Holder Inactive/Active modals
+  const [permitHolderModalData, setPermitHolderModalData] =
```

Comment:
Can we give this a more descriptive name like `permitHolderToUpdateStatus`. There aren't many good concise options here but it's best to make it related to the purpose of the modal (to update the status)

Reply:
Good call!